documentation/src/main/asciidoc/topics/con_performance_metric_considerations.adoc

```diff
@@ -2,7 +2,7 @@
 = Performance metric considerations
 {brandname} includes so many configurable combinations that determining a single formula for performance metrics that covers all use cases is not possible.  
 
-The purpose of the _{brandname} performance considerations and tuning guidelines_ document is to provide details about use cases and architectures that can help you determine requirements for your {brandname} deployment.
+The purpose of the _{PerformanceSize_docname}_ document is to provide details about use cases and architectures that can help you determine requirements for your {brandname} deployment.
```

Comment:
```suggestion
The purpose of the _{sizing_title}_ is to provide details about use cases and architectures that can help you determine requirements for your {brandname} deployment.
```
Think about both docs sets:
"Infinispan performance considerations and tuning guidelines" and  "Deployment Planning and Sizing Guide document"

Reply:
So the titles names are different from upstream and downstream? So I need ifdef statements?